internal/core/command/errors/types.go

```diff
@@ -26,3 +26,28 @@ func (e ErrCommandNotAssociatedWithDevice) Error() string {
 func NewErrCommandNotAssociatedWithDevice(commandID string, deviceID string) error {
 	return ErrCommandNotAssociatedWithDevice{commandID, deviceID}
 }
+
+type ErrExtractingInfoFromRequest struct {
+	commandID string
+	deviceID  string
+}
+
+func (e ErrExtractingInfoFromRequest) Error() string {
+	return fmt.Sprintf("error extracting command with id '%v' for device with id '%v'.", e.commandID, e.deviceID)
+}
+
+func NewErrExtractingInfoFromRequest(commandID string, deviceID string) error {
+	return ErrExtractingInfoFromRequest{commandID, deviceID}
+}
+
+type RequestParameterSupplied struct {
+	methodType string
+}
```

Comment:
Is this error struct needed? Can we simply use: https://github.com/edgexfoundry/edgex-go/blob/42aa357818f7845ac1db35d24861146831d3085c/internal/core/metadata/errors/types.go#L86-L88